react-native/react/constants/tracker.js

```diff
@@ -1,13 +1,21 @@
 /* @flow */
 
 // Simple state of the overall proof result
-export type SimpleProofState = 'normal' | 'warning' | 'error' | 'pending'
+export type SimpleProofState = 'normal' | 'warning' | 'error' | 'pending' | 'deleted' | 'checking'
```

Comment:
spoke to @malgorithms and @maxtaco about this. I'll make a map function to collapse things and we'll use the simple types that it collapses into